sapi/litespeed/lsapi_main.c

```diff
@@ -466,7 +466,7 @@ static sapi_module_struct lsapi_sapi_module =
 };
 /* }}} */
 
-static int init_request_info( TSRMLS_D )
+static void init_request_info( TSRMLS_D )
```

Comment:
This PR includes the last commit to our repository for some reason I don't know.
It's in the repository already. I don't get this diff by

git diff upstream/PHP-5.4

locally.